Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM node:6-stretch
+LABEL maintainer="will.price94@gmail.com"
+LABEL version="0.0.1"
+RUN apt-get update -qq && \
```

Comment:
a bit late, but was this here on accident?  there doesnt seem to be need for python

Reply:
No, it was intentional, surprisingly `npm install` fails as one of the deps (or transitive deps) requires python for building *sigh*.